### DISCUSSION TITLE
Set Default Quality Profiles for different users

### DISCUSSION THREAD

**Discussion by jasper-one** [seerr-team/seerr#197] [Ideas]
I think it would be great to set different quality profiles per user. For example, my partner can't tell the difference between 720p and 4k, so they are happy to take what they get. While other users only want blu-ray 1080p or above.

It would be great to know how to change the global default as well if someone knows how that works.

Thanks for your consideration :)

**Comment by fallenbagel** [comment]
We have been working on a filter system that achieves this c:
its just development been slow since everyone is busy :(
